integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -10,6 +10,7 @@ FROM connectedhomeip/chip-build-tizen:${VERSION} AS tizen
 FROM connectedhomeip/chip-build-crosscompile:${VERSION} AS crosscompile
 FROM connectedhomeip/chip-build-ameba:${VERSION} AS ameba
 FROM connectedhomeip/chip-build-k32w:${VERSION} AS k32w
+FROM connectedhomeip/chip-build-imx:${VERSION} AS imx
```

Comment:
since you are modifying an existing image (the vscode build) please bump the version file.

I tried to do this myself, however I failed because you seem to be working on master on your fork. For future PRs, consider  working of branches and that would allow some of us (me and justin have admin) to perform trivial patches and speed up PR merging.

Reply:
Thanks.
Actually version information is already link to the latest vension file. Now I can see all CI check passed. 

Also for create the PR from the master of my fork, I'm apologize for that. Will create specific branch name for the future PR.

Reply:
The file `chip-build/version` has to be changed with an increased number.

Right now it is 0.50 I believe. You have to change it to 0.51 and change the reason (we only use the reason to force merge conflicts on concurrent updates)

Reply:
Thank you for the detail guide for increasing the version number. I'm new to Docker so that have limit knowledge about and very appreciate for your help on this.
Now I have updated the patch and bump the version to 0.5.51 with the latest change, please help to review.